projects/plugins/jetpack/modules/widgets/milestone/class-milestone-widget.php

```diff
@@ -145,10 +149,10 @@ public static function styles_template() {
 }
 .milestone-countdown,
 .milestone-message {
-	background-color: <?php echo self::sanitize_color_hex( $colors['bg'] ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>;
-	border: 1px solid <?php echo self::sanitize_color_hex( $colors['border'] ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>;
+	background-color: var(--milestone-bg-color);
+	border: 1px solid var(--milestone-border-color);
 	border-top: 0;
-	color: <?php echo self::sanitize_color_hex( $colors['text'] ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>;
+	color: var(--milestone-text-color);
 	padding-bottom: 1em;
```

Comment:
Should these also be moved to the CSS file? (also, I did not highlight those were above). Was thinking that the only thing that the PHP end was gonna generate was the CSS var declarations.